### ISSUE TITLE
[Suggestion]/Request - Custom Directory Names

### ISSUE THREAD

**Issue by DWX12** [lumien231/Resource-Loader#2]
I know this sounds like the same issue that was closed and maybe it is, I'm not entirely sure. What I'm requesting is to be able to name directories whatever I want and either be able to tell Resource Loader where to look or what directory to use for specific mods, such as Custom Main Menu.

From your example, one of the root directories in the Resources folder is labeled test. I want my own custom folder name instead; however, Custom Main Menu will only look for the Test directory and nothing else.

One mod that allows you to specify what directory to look in is the Loading Screens mod by Blood N Bones Gaming group.

TLDR: I want to name my folders whatever I want to and tell Resource Loader where to look.

Thank you for the time and effort spent making this mod. You've helped out a ton of people looking to customize their Minecraft experience in addition to making modpacks look awesome.

**Comment by Liiw** [comment]
It seems that it will work if you use lowercase for your resource domain folder name. That is the folder located in the "resources" folder. I had a similar problem, when I wanted to use "CoT", but when using "cot" (and all the subfolders unchanged) it works.
